inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -226,17 +226,19 @@ export class AnimationCurveEditorComponent extends React.Component<
   }
 
   setValueLines() {
-    const initialValues = new Array(10).fill(0).map((s, i) => {
-      return { value: i * 10, label: -i };
+    const initialValues = new Array(10).fill(0).map((_, i) => {
+      return { value: i * 10, label: 2 * ((10 - i) / 10) };

```

Comment:
Is there a reason why this one is 2 * (10-i) / 10 and not `(10-i) / 5` ? It will be more consistent with the next calculation, which is `((i + 10) * -1) / 5`.

Reply:
Agree with Raanan and could you put the magic formulas in dedicated functions so that it helps with maintenance ???

Reply:
Yes, so the number 2, right now, is because by default we show the user values from 0 to 2 (for float type) on the screen. And the middle value is 1. The next calculation creates the values on the drag up or down of the screen. Yes, my plan is to separate (specify) this magic formula for other scales. 😄  So we eventually have (hopefully) one function and just send the proper ranges to recreate this one.

Reply:
Yup so would be cool to start creating it within this PR to enhance readibility.

Reply:
THX